Arrowgene.Ddon.GameServer/Handler/QuestGetRewardBoxHandler.cs

```diff
@@ -20,7 +30,30 @@ public QuestGetRewardBoxHandler(DdonGameServer server) : base(server)
 
         public override void Handle(GameClient client, IPacket packet)
         {
-            client.Send(GameFull.Dump_901);
+            // client.Send(GameFull.Dump_901);
+            S2CQuestGetRewardBoxListRes res = new S2CQuestGetRewardBoxListRes();
+
+            uint listNo = 1;
+            foreach (var boxReward in client.Character.QuestRewards)
+            {
+                var quest = QuestManager.GetQuest(boxReward.QuestId);
+                if (quest == null)
+                {
+                    // How did this happen?
+                    continue;
```

Comment:
That is just one quest in a list of rewards though. Maybe I can print out an error message instead?

Reply:
I'm not sure. Logging it would mean it would go unnoticed, as the player wouldn't see anything. It would maybe be better to return an error too so the player can report it when it happens and then an admin can check the logs for the source of the problem.